packages/connect/src/relay/index.ts

```diff
@@ -159,7 +160,10 @@ class Relay {
       return
     }
 
-    const handshakeResult = await new RelayHandshake(baseConnection.stream, this.options).initiate(relay, destination)
+    const handshakeResult = await new RelayHandshake(baseConnection.stream as any, this.options).initiate(
```

Comment:
Please split object instantiation, and awaiting initiation. This looks unwildly.

Reply:
you only added a type cast

Reply:
see https://github.com/hoprnet/hoprnet/pull/3688/commits/11829949152db098a952d292d5698d7063b15042